crates/common/tedge_config_macros/examples/multi.rs

```diff
@@ -0,0 +1,66 @@
+use camino::Utf8PathBuf;
+// use std::net::IpAddr;
```

Comment:
I need to remove this commented out code.

This will also become a more fleshed-out example in the future, this currently just checks that the new `multi` attribute doesn't cause compilation errors.